packages/widgetbook/lib/src/models/organizers/organizer_helper/story_helper.dart

```diff
@@ -11,7 +11,7 @@ class StoryHelper {
 
     final stories = List<WidgetbookUseCase>.empty(growable: true);
```

Comment:
why is it still named stories?

Reply:
because its an internal variable - will be renamed in another PR